go.mod

```diff
@@ -2,4 +2,12 @@ module github.com/zencoder/go-remote-config
 
 go 1.13
 
-require github.com/stretchr/testify v0.0.0-20151102014159-c478a808a1b3
+require (
+	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
+	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da
+	github.com/kr/text v0.2.0 // indirect
+	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 // indirect
+	github.com/stretchr/testify v1.8.4
+	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
+)
```

Comment:
Since the groupcache import was removed, perhaps we need a `go mod tidy` here to clean this up again?